doc/Pacemaker_Explained/en-US/Ch-Multi-site-Clusters.txt

```diff
@@ -347,8 +318,6 @@ you can run:
 
 == For more information ==
 
-`Multi-site Clusters`
-http://doc.opensuse.org/products/draft/SLE-HA/SLE-ha-guide_sd_draft/cha.ha.geo.html
+* http://doc.opensuse.org/products/draft/SLE-HA/SLE-ha-guide_sd_draft/cha.ha.geo.html[SUSE's Multi-site Clusters guide]
```

Comment:
Broken link ;-) The new link is here:
https://www.suse.com/documentation/sle-ha-geo-12/art_ha_geo_quick/data/art_ha_geo_quick.html

Other than these, it looks good to me.  Excellent work, thanks!

 @dmuhamedagic , you may want to take a look at it as well, in case I missed antyhing :-)